src/Main.elm

```diff
@@ -1,10 +1,198 @@
 module Main exposing (main)
 
-import Data.Audience
-import Data.AudienceFolder
-import Html exposing (Html)
+import Browser exposing (sandbox)
+import Data.Audience exposing (..)
+import Data.AudienceFolder exposing (..)
+import Dict exposing (Dict)
+import Dict.Extra as DE exposing (fromListDedupe)
+import Html exposing (Html, div, button, text)
```

Comment:
From `Html` I exposed only `Html, button, div, text` because I used only those (did not want to import the whole library). As for the `Html.Attributes.style`, I just forgot to import only `style`. Yes, it is a bit inconsistent, sorry. In the next update, I will expose only what I need (if the line, where I expose thing is not very long :D ).

Reply:
Well, in our codebase we use the full qualified names and don't import bare functions but only types. And for long module names we aliase them e.g.:

```elm
Html.div [Attrs.style "font-weight" "bold"] [Html.p [] [Html.text "wow, what a nice unicorn."] ]
```

this way we don't have the collisions when the names of functions in different modules are the same and we know exactly from where the function comes from without the need to look at the imports.

I think either way is fine

Reply:
Good idea 👍 What you suggested seems more clear.
The way how one imports a function does not impact a performance (yes, if you import the whole library with 100000+ functions, then yes). As there are (correct me if I am wrong) more ways to import a function in Elm, Elm developers may argue which way of importing a function is better and this may create a tension in a team if all of the members are too much enthusiastic and each of them prefers his own coding style. :D If the syntax for importing would be restricted to fewer options (or at least a linter could handle it) then so Elm developers could solve more of the more serious coding issues and not discuss coding styles so often. 
I always had a feeling while doing frontends in Elm, or Javascript, that developers discussed more coding styles than actual algorithms or more technically challenging issues. But, maybe it is only my experience/feeling.

Reply:
> I always had a feeling while doing frontends in Elm, or Javascript, that developers discussed more coding styles than actual algorithms or more technically challenging issues. But, maybe it is only my experience/feeling.

Certainly true. But honestly, there aren't as many actual algorithms to solve stuff with on the frontend side of things. You need to present stuff to user, let user act on it, etc., but there isn't much of processing etc. - you get all that mostly handed to you on a silver plate by the API.

There still are interesting things and optimizations, but overall I can't expect FE programming to be as algorithmically challenging as eg. data processing.

> If the syntax for importing would be restricted to fewer options (or at least a linter could handle it) then so Elm developers could solve more of the more serious coding issues and not discuss coding styles so often.

`elm-format` removes some of that but there still are some degrees of freedom where folks clash and have to come to some kind of compromise if they work on a shared codebase.